### PR TITLE
Remove incorrect references

### PR DESCRIPTION
**Pull request by riusricardo**
Remove incorrect references to Efinity Parachain.
The links are pointing to a forked chain that isn't connected to the Polkadot network.

### CODE REVIEW

**Review by al3mart — APPROVED** [review]
LGTM

File: packages/apps-config/src/links/subscan.ts [hunk 1]
```diff
@@ -33,7 +33,6 @@ export const Subscan: ExternalDef = {
     Dock: 'dock',
     'Dolphin Parachain Testnet': 'dolphin',
     Edgeware: 'edgeware',
-    Efinity: 'efinity',
```
Comment (jacogr) [inline packages/apps-config/src/links/subscan.ts:36]: This should not be removed.

**Review by jacogr — CHANGES_REQUESTED** [review]

File: packages/apps-config/src/endpoints/productionRelayPolkadot.ts [hunk 2]
```diff
@@ -204,21 +204,6 @@ export const prodParasPolkadot: Omit<EndpointOption, 'teleport'>[] = [
...
     }
   },
-  {
-    homepage: 'https://efinity.io',
-    info: 'efinity',
-    paraId: 2021,
-    providers: {
-      Dwellir: 'wss://efinity-rpc.dwellir.com',
-      Efinity: 'wss://rpc.efinity.io',
-      OnFinality: 'wss://efinity.api.onfinality.io/public-ws'
```
Comment (jacogr) [inline packages/apps-config/src/endpoints/productionRelayPolkadot.ts:214]: If incorrect for the parachain, and correct for the stand-alone chain, these providers should be moved to the production entry.

File: packages/apps-config/src/links/subscan.ts (hunk 1 above)
Comment (riusricardo) [reply packages/apps-config/src/links/subscan.ts:36]: Ok, will put it back.

File: packages/apps-config/src/endpoints/production.ts [hunk 3]
```diff
@@ -186,17 +186,6 @@ export const prodChains: Omit<EndpointOption, 'teleport'>[] = [
...
-    info: 'efinity',
-    providers: {
-      // Efinity: 'wss://rpc.efinity.io' // https://github.com/polkadot-js/apps/pull/6761
-    },
-    text: 'Efinity',
-    ui: {
-      color: '#496ddb',
-      logo: nodesEfinitySVG
-    }
-  },
```
[the remark this answers is not in the document]
Comment (riusricardo) [reply packages/apps-config/src/endpoints/production.ts:199]: I can also revert this change. However the RPC (that isn't enabled here) is pointing to a parachain fork connected to another relay-chain.
https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Frpc.efinity.io#/explorer

File: packages/apps-config/src/endpoints/productionRelayPolkadot.ts (hunk 2 above)
Comment (riusricardo) [reply packages/apps-config/src/endpoints/productionRelayPolkadot.ts:214]: But they took a recent decision to fork and start their own relay-chain and move the parachain to that chain.
https://twitter.com/enjin/status/1666818288616370177?t=6Hxo2-rutr_uKhK5buTRpw&s=19

Comment (jacogr) [reply packages/apps-config/src/endpoints/productionRelayPolkadot.ts:214]: Cool. I just read a bit about that this morning.

All good - however it doesn't change the fact that they do have a parachain on Polkadot. So chain 2021 is still Effinity, although -

- from their side it is not meant to be used
- we really don't support forked relays at all

So in this case, the best course of action is just to remove the providers. This means -

1. In the parachain listing paraId 2021 is still named Effinity
2. There is no avenue to actually connect to the specific chain (due to the 2 points above)

TL;DR Only remove the ability to connect.

**Comment by jacogr** [comment]
Thank you. I'm going to close this one in favor of https://github.com/polkadot-js/apps/pull/9561
